service/rpc2/server.go

```diff
@@ -571,9 +571,9 @@ func (c *RPCServer) AttachedToExistingProcess(arg AttachedToExistingProcessIn, o
 }
 
 type FindLocationIn struct {
-	Scope                     api.EvalScope
-	Loc                       string
-	IncludeNonExecutableLines bool
+	Scope api.EvalScope
+	Loc   string
+	Cfg   *api.FindLocCfg
```

Comment:
API changes need to be backwards compatible: you can't remove the IncludeNonExecutableLines parameter from here. Just add the FullPathMatch parameter.

Reply:
Yes, I do so because I think [ccf57b94](https://github.com/go-delve/delve/commit/ccf57b94#diff-ef4dee432332f81f2fbd37326df20715L5700) use api.FindLocCfg maybe better for compatible.   

I gree with you, I will just add FullPathMatch parameter for backwards compatible.  

Thx.